superdesk/vocabularies/vocabularies.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+
+import logging
+
+from flask import current_app as app
+from eve.utils import config
+
+from superdesk.resource import Resource
+from superdesk.services import BaseService
+from superdesk.utc import utcnow
+
+logger = logging.getLogger(__name__)
+
+
+class VocabulariesResource(Resource):
```

Comment:
The Schema definition is outdated. Please refer https://github.com/superdesk/superdesk/blob/master/server/apps/vocabularies/vocabularies.py#L43-L45